app/topics/shared/topic.model.ts

```diff
@@ -21,24 +21,44 @@ export class Topic {
   subTopics: Topic[];
   supervisors: User[];
   title: string;
-
+  studentID: number[];
+  supervisorID: number[];
+  reviewerID: number[];
```

Comment:
These are for the topic form. How can you send same 'userIds' to all the users?
(Consistency is being mentioned on the above comment.)
   ```
    data += this.userArrayJSON(this.reviewerID, 'Reviewers[]=');
    data += this.userArrayJSON(this.studentID, 'Students[]=');
    data += this.userArrayJSON(this.supervisorID, 'Supervisors[]=');
```